workers/unity/ProjectSettings/ProjectSettings.asset

```diff
@@ -157,7 +158,7 @@ PlayerSettings:
     tvOS: com.improbable.gdk
   buildNumber: {}
   AndroidBundleVersionCode: 1
-  AndroidMinSdkVersion: 16
+  AndroidMinSdkVersion: 21
```

Comment:
Is this intentional?

Reply:
I think I updated this to ensure Vulkan runs

Reply:
I guess it's not necessary any more since you were recommending not to use Vulkan as it's not great yet.